lib/daru/view/templates/googlecharts/static_html.erb

```diff
@@ -0,0 +1,7 @@
+<html lang='en'>
+<head>
+  <title>Chart</title>
```

Comment:
What if it is a google datatable ? I think we can add more information in title (id, chart type). We can extend it and display with table and chart both in html table ( mark as TODO. It must not be in this same PR).